### PR TITLE
Feature: Added `Description` property to `IAction`

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

**Notes**
* `Description` property _must_ be implemented in each action because actions that cannot be described should not exist. If we want to show or not show the description depending on the action or context, we should have a separate flag.
* For now, all descriptions other than copy, cut, and paste are placeholders. These should be replaced with appropriate descriptions before any additional features related to the command can be implemented. #11786 will be closed when all descriptions are implemented.

### CODE REVIEW

File: src/Files.App/Actions/IAction.cs [hunk 1]
```diff
@@ -5,17 +5,47 @@ namespace Files.App.Actions
 {
 	public interface IAction
 	{
+		/// <summary>
+		/// A label for display in context menus and toolbars
```
Comment (yaira2) [inline src/Files.App/Actions/IAction.cs:9]: It can be displayed in the command palette as well.

File: src/Files.App/Actions/IAction.cs [hunk 2]
```diff
@@ -5,17 +5,47 @@ namespace Files.App.Actions
...
+		/// Glyph information to display icon
+		/// </summary>
 		RichGlyph Glyph => RichGlyph.None;
 
+		/// <summary>
+		/// A hotkey to execute actions
+		/// </summary>
 		HotKey HotKey => HotKey.None;
+		/// <summary>
+		/// Another hotkey to execute actions
```
Comment (yaira2) [inline src/Files.App/Actions/IAction.cs:28]:
```suggestion
		/// Secondary hotkey to execute actions
```

File: src/Files.App/Actions/IAction.cs (hunk 1 above)
Comment (hishitetsu) [reply src/Files.App/Actions/IAction.cs:9]: I rather think that the command palette should display the description.
image

Comment (yaira2) [reply src/Files.App/Actions/IAction.cs:9]: 👍

**Review by yaira2 — APPROVED** [review]
LGTM

I think we should create the descriptions in a spreadsheet or table so that they can be reviewed side by side before we add them to the code.

**Comment by hishitetsu** [comment]
> I think we should create the descriptions in a spreadsheet or table so that they can be reviewed side by side before we add them to the code.

In specs directory?
https://github.com/files-community/Files/tree/main/specs

**Comment by hishitetsu** [comment]
OK. I'll make a table of actions.

**Pull request merged**